addon/resolver.js

```diff
@@ -25,6 +25,19 @@ import EmberResolver from 'ember-resolver';
 */
 export default EmberResolver.extend({
   /**
+    Contains names of type for which need lookup class with prefix received through a {{#crossLink "DeviceService"}}{{/crossLink}}.
```

Comment:
Names of types which needs to be resolved with prefixes received through a {{#crossLink "DeviceService"}}device service{{/crossLink}}